File: front/src/models/teamTask.js
class TeamTask {
    constructor({
        id,
        task_id: taskId,
        team_id: teamId,
        status,
        stolen,
        lost,
        score,
        checks,
        checks_passed: checksPassed,
        message,
    }) {
        this.id = id;
        this.taskId = taskId;
        this.teamId = teamId;
        this.status = status;
        this.stolen = stolen;
        this.lost = lost;
        this.sla = (100.0 * checksPassed) / Math.max(checks, 1);
        this.score = score;
        this.message = message;
    }

    static comp(A, B) {
        return A.taskId - B.taskId;
    }
}

export default TeamTask;

File: backend/lib/models/team_task.py
from .base import BaseModel


class TeamTask(BaseModel):
    team_id: int
    task_id: int
    status: int
    stolen: int
    lost: int
    score: float
    checks: int
    check_passed: int
    public_message: str
    private_message: str
    command: str
